relay/pkg/btc/remote/remote.go

```diff
@@ -0,0 +1,29 @@
+package remote
+
+import (
+	"math/big"
+
+	"github.com/ipfs/go-log"
+	"github.com/keep-network/tbtc/relay/pkg/btc"
+)
+
+var logger = log.Logger("relay-btc-remote")
+
+// remoteChain represents a remote Bitcoin chain.
+type remoteChain struct {
+	// TODO: implementation
+}
+
+// Connect connects to the Bitcoin chain and returns a chain handle.
+func Connect() (btc.Handle, error) {
```

Comment:
I'd consider exposing `bitcoin.Connect()` forwarding to this (then unexported) function. `remote.Connect()` is confusing, `bitcoin.Connect()` would look much better. And we can alias `bitcoin/local` as `btcLocal` for `btcLocal.Connect()` in tests.

Reply:
Sure we'll consider that during the actual implementations of chains connectivity. For now, it's just a starting point.

Reply:
On the other hand... import cycle. But yeah, let's see in the future.